cmd/strapi-demo/createdemodata/start.go

```diff
@@ -20,15 +20,15 @@ import (
 )
 
 const (
-	adminURLFlagName      = "admin-url"
-	adminURLFlagShorthand = "a"
-	adminURLFlagUsage     = "URL to run the strapi-demo instance on. Format: HostName:Port."
-	adminURLEnvKey        = "STRAPI-DEMO_ADMIN_URL"
-	adminURLEndpoint      = "/admin/auth/local/register"
-	studentCardsEndpoint  = "/studentcards"
-	transcriptEndpoint    = "/transcripts"
-	post                  = "post"
-	get                   = "get"
+	hostURLFlagName      = "host-url"
+	hostURLFlagShorthand = "u"
+	hostURLFlagUsage     = "URL to run the strapi-demo instance on. Format: HostName:Port."
+	hostURLEnvKey        = "STRAPI-DEMO_ADMIN_URL"
+	adminURLEndpoint     = "/admin/auth/local/register"
+	studentCardsEndpoint = "/studentcards"
+	transcriptEndpoint   = "/transcripts"
+	post                 = "POST"
```

Comment:
This was the main cause of the failer... post and get were in lower case so nothing was getting posted or retrieved